src/c++/perf_analyzer/request_rate_worker.h

```diff
@@ -64,6 +64,7 @@ class RequestRateWorker : public LoadWorker, public IScheduler {
     size_t seq_stat_index_offset_;
 
     bool is_paused_;
+    uint32_t num_of_sequences_;
```

Comment:
Can you move this above seq_stat_index_offset_? They are somewhat related so I'd prefer they were next to each other.

Reply:
Can we also rename to num_seq or num_sequences?  The "of" is weird to me. I know that is what the CLI option is, and I don't want us or anyone else to confuse this value with that CLI value.